### PR TITLE
feat(*): support mainnet with network flag

### PR DESCRIPTION
**Pull request by 0xHansLee**
Support mainnet with `--network` flag. The genesis.json and seeds.txt from Odyssey are used for now. We need to update this later. 

fyi, ref pr: #178 (which supports Odyssey)

issue: none

### CODE REVIEW

File: lib/netconf/mainnet/seeds.txt
Comment (jdubpark) [inline lib/netconf/mainnet/seeds.txt:1]: testnet seed values -> should we remove them for now and add seeds later?

**Review by jdubpark — APPROVED** [review]

File: lib/netconf/mainnet/seeds.txt
Comment (edisonz0718) [reply lib/netconf/mainnet/seeds.txt:1]: Yeah we should either use the new value or remove for now

Comment (0xHansLee) [reply lib/netconf/mainnet/seeds.txt:1]: Ok, I will remove it for now.

**Review by ezreal1997 — APPROVED** [review]

File: client/cmd/flags.go [hunk 1]
```diff
@@ -54,9 +54,9 @@ func bindInitFlags(flags *pflag.FlagSet, cfg *InitConfig) {
 }
 
 func bindValidatorBaseFlags(cmd *cobra.Command, cfg *baseConfig) {
-	cmd.Flags().StringVar(&cfg.RPC, "rpc", "https://odyssey.storyrpc.io", "RPC URL to connect to the testnet")
+	cmd.Flags().StringVar(&cfg.RPC, "rpc", "https://storyrpc.io", "RPC URL to connect to the testnet")
```
[the remark this answers is not in the document]
Comment (0xHansLee) [reply client/cmd/flags.go]: Done in [4e92007](https://github.com/piplabs/story/pull/424/commits/4e920072b6679e6ba758833ba56a0951aa81b29b).

**Pull request merged**